Automattic/jetpack
# Make Jetpack AI Paragraph block editable

Fixes https://github.com/Automattic/jetpack/issues/28384

Here's a video explaining proposed changes and possible regressions and improvements: 

https://www.loom.com/share/5993f814ab7b4edfb1b2f23ac004391c

### Proposed changes:

* Make the AI Paragraph block editable after AI content is generated


### Testing instructions:






* Add the AI Paragraph block to post
* Edit the content of the AI content that was generated
* Update or publish the post and verify the content is displayed as expected

### Possible Regression / Improvements

1. ~Typing forward slash `/` in the AI Paragraph block should display the block list. Once a block is chosen a new block should be created below the AI Paragraph block~ This is the typical behavior for blocks  ( except for the Paragraph block )
2. ~Creating an AI Paragraph block after another displays cached content.~ This was a cache issue 

### Does this pull request change what data or activity we track or use?
No
